src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -247,6 +247,7 @@ def create(
     options["num_styles"] = len(style_dataset)
     options["resnet_mlmodel_path"] = pretrained_resnet_model.get_model_path("coreml")
     options["vgg_mlmodel_path"] = pretrained_vgg16_model.get_model_path("coreml")
+    options["pretrained_weights"] = params["pretrained_weights"]
```

Comment:
Why is this needed?

Reply:
This is to allow the user to specify through the advanced parameters whether they want uniform initialization or initialization from the pre-trained model.